src/huggingface_hub/repository.py

```diff
@@ -270,6 +270,7 @@ def __init__(
         git_user: Optional[str] = None,
         git_email: Optional[str] = None,
         revision: Optional[str] = None,
+        private: Optional[bool] = None,
```

Comment:
Sorry to insist, by `private: bool = False` seems strictly better.

The `None` is necessary when the default argument is mutable, other it's better to use a real default, no ?